package.json

```diff
@@ -39,7 +39,8 @@
     "test": "lerna run --stream test",
     "test.watch": "lerna run --stream test.watch",
     "ci-load": "node ./packages/test/lib/load/all-in-one.js --server-address $TEMPORAL_TESTING_SERVER_URL --iterations 10000 --otel-url='' --max-cached-wfs 500",
-    "ci-nightly": "node --trace-exit ./packages/test/lib/load/all-in-one.js --server-address $TEMPORAL_TESTING_SERVER_URL --for-seconds 14400 --min-wfs-per-sec 5 --concurrent-wf-clients 10 --otel-url=''",
+    "ci-nightly-longrun": "node ./packages/test/lib/load/all-in-one.js --server-address $TEMPORAL_TESTING_SERVER_URL --for-seconds 14400 --min-wfs-per-sec 5 --concurrent-wf-clients 10 --otel-url=''",
```

Comment:
nit: I'd still prefer `--otel-url` to default to undefined in the code and omit it here.